src/locales/default.json

```diff
@@ -576,6 +576,7 @@
   "noPriceImpact": "Best price",
   "noPriceImpactTip": "Adding tokens proportionally gets you the best price.",
   "noPriceInfo": "Price information is missing for this pool, since it contains a token not found by our price provider.",
+  "nonAllowedRateProviders": "One or more token price providers for this pool have not been approved by Balancer. Proceed with caution.",
```

Comment:
cc @pkattera in case you'd like to chime in on this copy

Reply:
I spoke with Markus about this: Let's change this to:
`One or more token rate providers associated with tokens in this pool have not been vetted. Proceed with caution.`

Reply:
```suggestion
  "nonAllowedRateProviders": "One or more token rate providers associated with tokens in this pool have not been vetted. Proceed with caution.",
```